share/src/main/java/io/zensoft/share/service/DefaultVacancyRetriever.java

```diff
@@ -0,0 +1,14 @@
+package io.zensoft.share.service;
+
+import io.zensoft.share.model.Vacancy;
+import io.zensoft.share.model.VacancyResponse;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DefaultVacancyRetriever implements VacancyRetriever {
```

Comment:
recommend you to rename this class from `DefaultVacancyRetriever` to `DefaultVacancyRetrieverService`